ethereumetl/cli/export_all.py

```diff
@@ -113,12 +116,13 @@ def get_partitions(start, end, partition_batch_size, provider_uri):
               help='The URI of the web3 provider e.g. '
                    'file://$HOME/Library/Ethereum/geth.ipc or https://mainnet.infura.io')
 @click.option('-o', '--output-dir', default='output', show_default=True, type=str, help='Output directory, partitioned in Hive style.')
+@click.option('-P', '--postgres-connection-string', default='', show_default=False, type=str, help='Postgres connection string.')
```

Comment:
Is this the same format they use for the stream command?

Reply:
They overload the output option to be either a directory or a connection string - we want to dual write so making the postgres option explicit